lib/ChangeDueDateDialog/ChangeDueDateDialog.js

```diff
@@ -298,6 +301,7 @@ class ChangeDueDateDialog extends React.Component {
     const openRequests = get(this.props, 'resources.openRequests.records', []);
 
     const requestCounts = openRequests.reduce((requests, record) => {
+      if (record.status === RequestCancelledStatus) { return requests; }
```

Comment:
```suggestion
      if (record.status === RequestCancelledStatus) return requests;
```
I like how you avoid counting canceled requests. Just a tiny bit of style advice. We usually don't use curly braces in a project when we return a single-word expression.